docs/rfcs/XXX-pull-request-review.md

```diff
@@ -18,69 +18,126 @@ Peer review is also a critical part of the path to acceptance for pull requests
-* Clicking a line comment opens or activates an editor on the referenced file and scrolls to center the comment's line, translated according to local changes if appropriate.
-* Line comments within the review are rendered: _with a vertical blue bar_ before the file has been opened and the corresponding decoration is visible; _with a vertical grey bar_ after the file and decoration have been seen; and _with a vertical green bar_ after the comment has been marked "resolved" with the control on its decoration.
-* The review summary bubble and line comment lists are greyed out if a different `IssueishDetailItem` is activated.
+Clicking a pull request in the list opens a `PullRequestDetailItem` in the workspace center.
 
-If a pending review is present, it appears at the top of the "Reviews" section within this tile:
+For PRs that are not listed in the panel, users can use the `github:open-issue-or-pull-request` command:
 
-![pending-review](https://user-images.githubusercontent.com/378023/46275946-9bd69680-c599-11e8-9889-66c35458286a.png)
+<img width="679" alt="xxx-pull-request-review_md_ ___github_github" src="https://user-images.githubusercontent.com/7910250/46391629-c8baa480-c692-11e8-8010-0cceb69394b1.png">
+
+
+### PullRequestDetailItem
+
+Each `PullRequestDetailItem` opened on a pull request displays the full, multi-file diff associated with the pull request. Review comments are shown within the diff. See ["Comment decorations"](#comment-decorations) for description of review comments.
+
+![screen shot 2018-10-03 at 1 50 18 pm](https://user-images.githubusercontent.com/7910250/46391711-1df6b600-c693-11e8-87f3-ad4cdbe8ebd8.png)
+
+TODO: update mock to have "Start review button" and "Add single comment"
+
+Diffs are editable ONLY if the pull request branch is checked out and the local branch history has not diverged from the remote branch history. Otherwise diffs are not editable.
+
+A panel at the bottom of the pane offers various options for sorting and filtering the diff. It also has a "Review Changes" button.
+
+#### Sort Options
+
+<img width="731" alt="slack_-_github" src="https://user-images.githubusercontent.com/7910250/46392358-f6551d00-c695-11e8-8ed4-c7aa95044b06.png">
+
+The default view is sorted by files. This is akin to the "Files changed" tab on dotcom. It displays the diff for all changed files in the PR.
+
+Sorting by reviews is akin to the review summaries that appear on the "Conversation" tab on dotcom. The comments are displayed grouped by review along with some context lines.
+
+![screen shot 2018-10-03 at 1 50 08 pm](https://user-images.githubusercontent.com/7910250/46394598-6ebfdc00-c69e-11e8-84eb-39ccbcccf736.png)
+
+TODO: include show multiple reviews stacked
+
+Sorting by commits is akin to the "Commits" tab on dotcom. A list of commits is displayed in chronological order, oldest commit on top. Clicking a commit expands the diff contents below. If there is a commit message body this is displayed as well.
```

Comment:
I was imagining that you could not edit commit diffs. Viewing commits would show you exactly the changes introduced in that commit and that's it.

> If not, we have a situation where you could start an edit in "review mode," change a radio button, and... what happens to your edits then?

This is a good question. I'm imagining that the edits remain there, and when you switch back to "review mode" you see them exactly as you left them. Like, way of implementing this could be to keep the review mode component untouched and simply show the "commit mode" component on top of it. And in this mode, you wouldn't necessarily expect to see the edits you made because you're viewing commit changes. 

Does that address your concern? I might not be fully understanding the question because I don't see how this relates to whether or not the commit diffs are editable.